examples/multiple_services/README.md

```diff
@@ -139,6 +139,7 @@ spec:
     version: v1alpha1
     kind: Database
     name: db-demo
+    namingStrategy: '{{ .service.kind | upper }}_{{ .name | upper }}'
```

Comment:
After reading this more than twice, wouldn't it be the case to use this as default naming strategy? If this is not the case, then examples should be adjusted to consume the default naming strategy.

Reply:
no need to define this explicitly, since this strategy is going to be default.